sklearn/feature_selection/rfe.py

```diff
@@ -65,6 +66,15 @@ class RFE(BaseEstimator, MetaEstimatorMixin, SelectorMixin):
         If within (0.0, 1.0), then ``step`` corresponds to the percentage
         (rounded down) of features to remove at each iteration.
 
+    step_one_threshold : int or None (default=None)
+        If specified step int or float equates to > 1 feature then the
```

Comment:
This is not very clear.

Reply:
Yes I agree that’s why I need feedback! 

I’ve had my own customized RFE for over a year with these features and they are features that are described by Guyon et al in the original SVM-RFE paper.

I am totally open to param name changes etc. The param here is where you set a threshold eg 200 where the RFE starts stepping by one feature at a time again if you’ve set step equal to an int or float that corresponds to stepping multiple features at a time. At some threshold num features depending on your problem you want to start stepping one at a time.

Reply:
I think the effect of this is that "Feature sets sized between `n_features_to_select` and `step_one_threshold`, inclusive, will all be considered regardless of the `step` parameter."

Possible names:
* `try_all_below`
* `step_if_at_least`